__tests__/gui/data_mapper.feature

```diff
@@ -49,27 +50,9 @@ Feature: Data Mapper
     And I expand Data Mapper
     Then I check if there are 3 categories
 
-    And I navigate to WC and back to ZA in 100 ms
```

Comment:
removed these because this is not possible to re-create anymore so these test steps were irrelevant